src/components/CameraControls/index.tsx

```diff
@@ -151,7 +151,7 @@ const CameraControls = ({
                 >
                     <Button
                         className={styles.btn}
-                        icon={Icons.ZoomIn}
+                        icon={ZoomIn}
                         onClick={zoomIn}
```

Comment:
Nit: Should these buttons have an `aria-label` or other readable text label, or is that included with the icon itself?

Reply:
Great point, fits better in the next set of changes, noted!